css/buttons.css

```diff
@@ -0,0 +1,90 @@
+/* GENERAL */
+
+.button-div {
+    margin-top: 1;
+    margin-bottom: 15px;
+}
+
+/* TOGGLE SWITCH FOR HIDING ELEMENTS */
+
+input[type=checkbox] {
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    -webkit-tap-highlight-color: transparent;
+    cursor: pointer;
+  }
+  input[type=checkbox]:focus {
+    outline: 0;
+  }
+
+  input,
+  label{
+  vertical-align:middle;
+}
+  
+  .toggle {
+    height: 32px;
+    width: 52px;
+    border-radius: 16px;
+    display: inline-block;
+    position: relative;
+    margin: 0;
+    border: 2px solid #474755;
+    background: linear-gradient(180deg, #2D2F39 0%, #1F2027 100%);
+    transition: all 0.2s ease;
+  }
+  .toggle:after {
+    content: "";
+    position: absolute;
+    top: 2px;
+    left: 2px;
+    width: 24px;
+    height: 24px;
+    border-radius: 50%;
+    background: white;
+    box-shadow: 0 1px 2px rgba(44, 44, 44, 0.2);
+    transition: all 0.2s cubic-bezier(0.5, 0.1, 0.75, 1.35);
+  }
+  .toggle:checked {
+    border-color: #654FEC;
+  }
+  .toggle:checked:after {
+    transform: translatex(20px);
+  }
+  
+
+
+/* SAVE IMAGE BUTTON */
+
+/* Button and the icon */
+
+.trigger {
+	position: relative;
+	background: #4a1942;
+	display: inline-block;
+	width: 200px;
+	height: 60px;
+	border-radius: 50em;
+	box-shadow: 0 2px 20px rgba(0,0,0,0.7), inset 0 1px 0 rgba(255,255,255,0.2);
+	color: #fff;
+}
+
+span, strong {
+	position: absolute;
+	width: 100%;
+	height: 100%;
+	left: 0;
+}
+
+strong {
```

Comment:
same here. you can even do `.button-trigger strong` if you don't want to add more classes (not too many classes is a good thing!)